website_event_filter_organizer/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# Copyright 2019 Tecnativa - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
```

Comment:
Add `website` key: `https://github.com/OCA/website`

Reply:
I'm seeing now that I said this should go to OCA/event, and it should still be moved.

Reply:
@sergio-teruel then URL will change `https://github.com/OCA/event`